crypto/keyring/legacy_info.go

```diff
@@ -187,68 +193,68 @@ type multisigPubKeyInfo struct {
 }
 
 // multiInfo is the public information about a multisig key
-type multiInfo struct {
+type LegacyMultiInfo struct {
```

Comment:
Just wondering since `LegacyMultiInfo` is a public struct, should `legacyInfo` be public interface which is implemented by `LegacyMultiInfo`?